lms/djangoapps/courseware/grades.py

```diff
@@ -543,16 +563,17 @@ def _progress_summary(student, request, course, field_data_cache=None, scores_cl
                     if correct is None and total is None:
                         continue
 
-                    scores.append(
-                        Score(
-                            correct,
-                            total,
-                            graded,
-                            module_descriptor.display_name_with_default,
-                            module_descriptor.location
-                        )
+                    score = Score(
+                        correct,
+                        total,
+                        graded,
+                        module_descriptor.display_name_with_default,
+                        module_descriptor.location
                     )
 
+                    scores.append(score)
+                    locations_to_scores[module_descriptor.location] = score
```

Comment:
It's possible that I don't understand what you mean by raw and weighted - the scores being passed around are Score objects which give both the earned and the possible scores. Would you consider that to be raw?